### DISCUSSION TITLE
Sorry y'all but I'm new and teaching myself to coding. Lost?

### DISCUSSION THREAD

**Discussion by Jcrowell74** [vercel/next.js#18466] [Help]
I am lost with this and looking for direction. I started to look for a repository, but IDK how?

**Comment by pierrenel** [comment]
@Jcrowell74 start here: https://nextjs.org/learn/basics/create-nextjs-app and good luck! 👍

**Reply by Jcrowell74** [reply]
Thx for your advise. Many thx
